doc/config.md

```diff
@@ -1226,6 +1226,11 @@ they are logged but ignored otherwise.
 Set the pool mode specific to this database. If not set,
 the default `pool_mode` is used.
 
+### max_db_client_connections
```

Comment:
nit: let's move this below max_db_connections

Reply:
Done. Was thinking about the naming convention. Currently we are using 
max_db_client_connections/max_db_connections. Perhaps we should change it to max_db_client_connections/max_db_server_connections?

Reply:
Let's keep as is.

Reply:
understood.